spec/rack/test/uploaded_file_spec.rb

```diff
@@ -41,7 +41,7 @@ def test_file_path
         require 'java'
         java_import 'java.lang.System'
 
-        20.times do |i|
+        50.times do |i|
```

Comment:
Yes, this part is to pass the Travis test. OK I send another PR for that.
I forgot this repository's rule. :)